tests/AutomaticModeTest.php

```diff
@@ -2,134 +2,111 @@
-    }
+    expect(app('tenancy_initialized_for_tenant'))->toBe('foobar');
+});
 
-    /** @test */
-    public function central_helper_runs_callbacks_in_the_central_state()
-    {
-        tenancy()->initialize($tenant = Tenant::create());
+test('central helper runs callbacks in the central state', function () {
+    tenancy()->initialize($tenant = Tenant::create());
 
-        tenancy()->central(function () {
-            $this->assertSame(null, tenant());
-        });
+    tenancy()->central(function () {
+        expect(tenant())->toBe(null);
+    });
 
-        $this->assertSame($tenant, tenant());
-    }
+    expect(tenant())->toBe($tenant);
+});
 
-    /** @test */
-    public function central_helper_returns_the_value_from_the_callback()
-    {
-        tenancy()->initialize(Tenant::create());
+test('central helper returns the value from the callback', function () {
+    tenancy()->initialize(Tenant::create());
 
-        $this->assertSame('foo', tenancy()->central(function () {
-            return 'foo';
-        }));
-    }
+    $this->assertSame('foo', tenancy()->central(function () {
+        return 'foo';
+    }));
+});
 
-    /** @test */
-    public function central_helper_reverts_back_to_tenant_context()
-    {
-        tenancy()->initialize($tenant = Tenant::create());
+test('central helper reverts back to tenant context', function () {
+    tenancy()->initialize($tenant = Tenant::create());
 
-        tenancy()->central(function () {
-            //
-        });
+    tenancy()->central(function () {
+        //
+    });
 
-        $this->assertSame($tenant, tenant());
-    }
+    expect(tenant())->toBe($tenant);
+});
 
-    /** @test */
-    public function central_helper_doesnt_change_tenancy_state_when_called_in_central_context()
-    {
-        $this->assertFalse(tenancy()->initialized);
-        $this->assertNull(tenant());
+test('central helper doesnt change tenancy state when called in central context', function () {
+    expect(tenancy()->initialized)->toBeFalse();
+    expect(tenant())->toBeNull();
 
-        tenancy()->central(function () {
-            //
-        });
+    tenancy()->central(function () {
+        //
+    });
 
-        $this->assertFalse(tenancy()->initialized);
-        $this->assertNull(tenant());
-    }
+    expect(tenancy()->initialized)->toBeFalse();
+    expect(tenant())->toBeNull();
+});
+
+// Helpers
+function bootstrap(\Stancl\Tenancy\Contracts\Tenant $tenant)
+{
+    app()->instance('tenancy_initialized_for_tenant', $tenant->getTenantKey());
 }
 
-class MyBootstrapper implements TenancyBootstrapper
+function revert()
 {
-    public function bootstrap(\Stancl\Tenancy\Contracts\Tenant $tenant)
-    {
-        app()->instance('tenancy_initialized_for_tenant', $tenant->getTenantKey());
-    }
-
-    public function revert()
-    {
-        app()->instance('tenancy_ended', true);
-    }
+    app()->instance('tenancy_ended', true);
```

Comment:
The original class should be added back in https://github.com/archtechx/tenancy/blob/3.x/tests/AutomaticModeTest.php#L124-L135

Reply:
And in this case, I added the back `MyBootstrapper` class. https://github.com/archtechx/tenancy/blob/shift-64622/tests/AutomaticModeTest.php#L108-L119